plugins/renepay/pay.c

```diff
@@ -649,14 +706,17 @@ payment_listsendpays_previous(
 
 	json_for_each_arr(i, t, arr)
 	{
-		u64 partid, groupid;
-		struct amount_msat this_msat, this_sent;
+		u64 partid = 0, groupid;
+		struct amount_msat this_msat = AMOUNT_MSAT(0), this_sent;
 		const char *status;
```

Comment:
Are these dummy initializations really needed? It prevents `valgrind` from tracking the initialization state of memory.

Reply:
I see the need to `partid` below, however the `this_msat` variable may remain uninitialized.

Reply:
Fixed c71628